src/test/scala/ai/privado/exporter/FeignEgressExportTest.scala

```diff
@@ -23,22 +23,47 @@
 
 package ai.privado.exporter
 
-import ai.privado.cache.{AppCache, S3DatabaseDetailsCache}
+import ai.privado.cache.{AppCache, RuleCache, S3DatabaseDetailsCache}
 import ai.privado.entrypoint.PrivadoInput
 import ai.privado.exporter.HttpConnectionMetadataExporter
 import ai.privado.languageEngine.java.JavaTaggingTestBase
 import ai.privado.tagger.sink.RegularSinkTagger
+import ai.privado.testfixtures.JavaFrontendTestSuite
+import ai.privado.model.{CatLevelOne, ConfigAndRules, Constants, FilterProperty, Language, NodeType, RuleInfo}
 
 import scala.collection.mutable
 
-class FeignEgressExportTest extends JavaTaggingTestBase {
+class FeignEgressExportTest extends JavaFrontendTestSuite {
 
-  override def beforeAll(): Unit = {
-    super.beforeAll()
-  }
+  val ruleCache = new RuleCache()
+  val appCache  = new AppCache()
+  appCache.repoLanguage = Language.JAVA
```

Comment:
No need to set appCache explicitly it is already part of JavaFrontendTestSuite

Reply:
Yes right.